src/app/submission/components/submit/submit.component.html

```diff
@@ -85,15 +85,9 @@ <h3 class="vf-u-margin__top--xxl vf-text vf-text-heading--4">What's next?</h3>
   </button>
 </ng-container>
 
-
-<ng-container *ngIf="exportLink">
-  <button mat-raised-button color="primary" (click)="requestExport()">Submit to HCA</button>
-</ng-container>
-
```

Comment:
It can happen that a submission will only have export link and not submit link (E.g. if the submission state is Archived) 

So, I think we need to figure out why the submit link and export link are being available at the same time. I think the old behaviour is that only submit link is available when the submission state is Valid, and the export link will only appear if the state is in Archived. If you remove this, there'll be no way to trigger exporting from the Archived state. 
Example Archived submission in prod: https://contribute.data.humancellatlas.org/submissions/detail?uuid=76283647-9b00-4651-9224-18a4db2b7b29

Reply:
The submission does still go to "Archiving" using this method, and it still generates the bundle manifests (assays tab)

https://user-images.githubusercontent.com/13031259/150171431-3318de2c-36d9-43fa-9328-7e9db27db274.mov

.

Reply:
If you push these changes there will be no buttons displayed when submission is in _Archived_ state. There should be an export action available if the submission is in _Archived_ state.

Reply:
The buttons are being displayed depending on the links available. I was saying that there should be no 'export' link in the _links of the submission json if it's in _Graph Valid_ state.